README.md

```diff
@@ -3,49 +3,75 @@
 
-NowSecure quickly identifies and details real issues, provides remediation recommendations, and integrates with ticketing systems such as Gitlab and Jira.
+NowSecure quickly identifies and details real issues, provides remediation recommendations, and integrates with ticketing systems such as Azure DevOps and Jira.
 
 This integration requires a NowSecure platform license. See <https://www.nowsecure.com> for more information.
 
-## Job Parameters
+## Getting Started
+
+### Installation
+
+First, find this extension in the [Azure DevOps Marketplace](https://marketplace.visualstudio.com/items?itemName=Nowsecure-com.nowsecure-azure-ci-extension)
+
+Then install it following [Microsoft's instructions](https://learn.microsoft.com/en-us/azure/devops/marketplace/install-extension?view=azure-devops) on installing Azure DevOps Marketplace extensions.
+
+**NOTE:** Currently, compatibility is limited to either Windows / Linux running an X64 architecture, or MacOS on ARM. In order for the extension to work, please make sure you are running on an appropriate `vmImage`.
+
+### Configuration 
 
 To add this component to your CI/CD pipeline, the following should be done:
 
-- Get a token from your NowSecure platform instance. More information on this can be found in the [NowSecure Support Portal](https://support.nowsecure.com/hc/en-us/articles/7499657262093-Creating-a-NowSecure-Platform-API-Bearer-Token)
+- Get a token from your NowSecure platform instance. More information on this can be found in the [NowSecure Support Portal](https://support.nowsecure.com/hc/en-us/articles/7499657262093-Creating-a-NowSecure-Platform-API-Bearer-Token).
 - Identify the ID of the group in NowSecure Platform that you want your assessment to be included in. More information on this can be found in the
   [NowSecure Support Portal](https://support.nowsecure.com/hc/en-us/articles/38057956447757-Retrieve-Reference-and-ID-Numbers-for-API-Use-Task-ID-Group-App-and-Assessment-Ref).
-  (Note: Authentication is required to access this page)
-- Add a CI/CD variable to your project named, `NS_TOKEN` and add the token created above. As this is a credential, be sure to check the box labeled `Keep this value secret`.
-- Add the following include entry to your project's CI/CD configuration and set your input values
+- Add an [Azure DevOps Secret Variable](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/set-secret-variables?view=azure-devops&tabs=yaml%2Cbash#secret-variable-in-the-ui) to your project named, `NS_TOKEN`. Set this to the value of the token created above. 
+- Add the following include entry to your project's CI/CD configuration and set your input values:
 
   ```yaml
   - task: nowsecure-azure-ci-extension@<tag>
     inputs:
+      # Required Inputs    
       group: "<group-ref>"
       token: $(NS_TOKEN)
       binary_file: "<path-to-binary>"
+      # Recommended Optional Inputs
+      minimum_score: 70
```

Comment:
I would remove `minimum_score` if we're telling them to copy/paste this code block to get it working

Reply:
Well, I guess it does say recommended optional...

I do think that an initial minimum_score might lead to higher error rates than they'd want tho. 70 is very high particularly for android apps 😬

Reply:
Leaving as is to match past extension's config